Source/WebKit/Shared/Cocoa/ArgumentCodersCocoa.mm

```diff
@@ -477,6 +478,7 @@ static bool shouldEnableStrictMode(Decoder& decoder, const HashSet<Class>& allow
             || messageName == IPC::MessageName::AuthenticationManager_CompleteAuthenticationChallenge; // UIP -> NP
     };
 
+#if ENABLE(DATA_DETECTION) || ENABLE(REVEAL)
```

Comment:
I don't understand why this is needed now but not before.

Reply:
To fix the watchOS build. The "newer OS-es shortcut" at the top of `shouldEnableStrictMode` is now evaluating to false for watchOS 10, while we're using `HAVE(SECURE_ACTION_CONTEXT)` outside of `ENABLE(DATA_DETECTION)`, which broke the initial assumption under which that code was introduced...

```cpp
#if ((PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 140000) \
    || ((PLATFORM(IOS) || PLATFORM(MACCATALYST)) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 170000)) \
    || PLATFORM(VISION)
// Only used inside ENABLE(DATA_DETECTION)
#define HAVE_SECURE_ACTION_CONTEXT 1
#endif
```